src/components/organisms/NavBar.tsx

```diff
@@ -0,0 +1,67 @@
+import Link from 'next/link'
+import { useState } from 'react'
+
+ const NavBar = () => {
```

Comment:
"function" key word for functional components
```suggestion
function NavBar() {
```